crates/duty-tracker/src/contract_state_machine.rs

```diff
@@ -291,12 +294,18 @@ pub enum OperatorDuty {
     PublishRootNonce {
         /// Transaction ID of the DRT
         deposit_request_txid: Txid,
+
+        /// The schnorr key that locks the funds in the DRT for the user to take back after a
```

Comment:
The key is not Schnorr.
The signing is Schnorr.
The key is an X-coordinate in the secp256k1.
```suggestion
        /// The X-only public key that locks the funds in the DRT for the user to take back after a
```